### DISCUSSION TITLE
How to Know if a CCD Encounter is Inpatient or Outpatient

### DISCUSSION THREAD

**Discussion by JJWest12** [nextgenhealthcare/connect#5044] [Q&A]
I have been looking at some CCDs and doing some research online to try and see how someone would know the difference between an inpatient or outpatient CCD but I haven't been able to find much. Is the only way to know is by looking at the encounter time? Or is there something else somewhere in the CCD that specifically will tell you if the encounter is inpatient or outpatient?

**Comment by rjkroll** [comment]
Well, like you already know, the document itself can contain both Inpatient and Outpatient information, without an actual way to really designate the whole document as an "Inpatient CCD" or "Outpatient CCD".   

Here are some sections, though, that usually only pertain to Inpatient data, so you can "probably" assume that if the document contains these sections, then it is for an inpatient:

Hospital Course (inpatient visits only) 
Reason for Hospitalization (inpatient visits only)
Hospital Discharge Instructions (displayed for observation [O]and day surgery [S] visits, in addition to inpatient visits)

Additionally, you could probably using the Encounter time tag to determine it, assuming the sending organization sends the proper "start/end" time/dates of those tags
